cmd/syncthing/main.go

```diff
@@ -230,7 +230,7 @@ func parseCommandLineOptions() RuntimeOptions {
 	flag.BoolVar(&options.Verbose, "verbose", false, "Print verbose log output")
 	flag.BoolVar(&options.paused, "paused", false, "Start with all devices and folders paused")
 	flag.BoolVar(&options.unpaused, "unpaused", false, "Start with all devices and folders unpaused")
-	flag.StringVar(&options.logFile, "logfile", options.logFile, "Log file name (still always logs to stdout).")
+	flag.StringVar(&options.logFile, "logfile", options.logFile, "Log file name (still always logs to stdout). This is best effort, i.e. Syncthing will keep running if encountering errors writing to this file.")
```

Comment:
I don't think we need to be this verbose really...

<img width="829" alt="Screenshot 2020-02-26 at 07 43 14" src="https://user-images.githubusercontent.com/125426/75318874-bdf64600-586b-11ea-872e-6855c121a429.png">